Holovibes/sources/thread/information_worker.cc

```diff
@@ -308,14 +318,16 @@ void InformationWorker::display_gui_information()
 
     for (auto const& [key, value] : FastUpdatesMap::map<QueueType>)
     {
-        if (key == QueueType::UNDEFINED)
+        if (key == QueueType::UNDEFINED || api::get_import_type() == ImportType::None)
```

Comment:
You can put the new condition before the loop instead of checking it for every queue

Reply:
i agree with that